README.md

```diff
@@ -46,42 +46,102 @@ Run `test_runner/flank.ios.yml` with flank to verify iOS execution is working.
+  # record-video: true
+
+  ## The max time this test execution can run before it is cancelled (default: 15m).
+  ## It does not include any time necessary to prepare and clean up the target device.
+  ## The maximum possible testing time is 30m on physical devices and 60m on virtual devices.
+  ## The TIMEOUT units can be h, m, or s. If no unit is given, seconds are assumed.
+  # timeout: 30m
+
+  ## Invoke a test asynchronously without waiting for test results.
+  # async: false
+
+  ## The billing enabled Google Cloud Platform project name to use
   # project: delta-essence-114723
-  # results-history-name: ios-history
 
-  # test and xctestrun-file are the only required args
+  ## The history name for your test results (an arbitrary string label; default: the application's label from the APK manifest).
+  ## All tests which use the same history name will have their results grouped together in the Firebase console in a time-ordered test history list.
+  # results-history-name: android-history
+
+  ## Experimental!
+  ## The number of times a TestExecution should be re-attempted if one or more\nof its test cases fail for any reason.
+  ## The maximum number of reruns allowed is 10. Default is 0, which implies no reruns.
+  # flaky-test-attempts: 0
+
+  # -- IosGcloudYml --
+
+  ## The path to the test package (a zip file containing the iOS app and XCTest files).
+  ## The given path may be in the local filesystem or in Google Cloud Storage using a URL beginning with gs://.
+  ## Note: any .xctestrun file in this zip file will be ignored if --xctestrun-file is specified.
   test: ./src/test/kotlin/ftl/fixtures/tmp/EarlGreyExample.zip
+
+  ## The path to an .xctestrun file that will override any .xctestrun file contained in the --test package.
+  ## Because the .xctestrun file contains environment variables along with test methods to run and/or ignore,
+  ## this can be useful for customizing or sharding test suites. The given path should be in the local filesystem.
   xctestrun-file: ./src/test/kotlin/ftl/fixtures/tmp/EarlGreyExampleSwiftTests_iphoneos12.1-arm64e.xctestrun
-  xcode-version: 9.2
-  device:
-    - model: iphone8
-      version: 11.2
-      locale: en
-      orientation: portrait
-  # The number of times to retry failed tests. Default is 0. Max is 10.
-  flaky-test-attempts: 0
+
+  ## The version of Xcode that should be used to run an XCTest.
+  ## Defaults to the latest Xcode version supported in Firebase Test Lab.
+  ## This Xcode version must be supported by all iOS versions selected in the test matrix.
+  # xcode-version: 10.1
+
+  ## A list of DIMENSION=VALUE pairs which specify a target device to test against.
+  ## This flag may be repeated to specify multiple devices.
+  ## The four device dimensions are: model, version, locale, and orientation.
+  # device:
+  #  - model: iphone8
+  #   version: 12.0
+  #   locale: en
+  #   orientation: portrait
```

Comment:
why don't you add a second one here, so everyone knows how to work with more than one device?

```
      - model: iphonex
        version: 12.0
        locale: es_ES
        orientation: landscape
```

Reply:
great idea, done :)